ts/packages/cli/src/helpers/synchronizers/logic/Netlify.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import { EvaluatedConfigs } from '@configu/ts';
+import { CONFIG_SYNCHRONIZER_LABEL } from '@configu/lib';
+
+const label = CONFIG_SYNCHRONIZER_LABEL.Netlify;
+
+export type NetlifyConfiguration = {
+  // * Netlify docs: https://docs.netlify.com/configure-builds/environment-variables/
+  token: string;
+  siteId: string;
+};
+
+// TODO - Decide between syncNetlifyConfigs (previous name) and syncNetlifyEnvVars
+export const syncNetlifyEnvVars = async ({
```

Comment:
@rannn505 We previously named this `syncNetlifyConfigs`, Netlify names their 'configs' as 'environment variable' so I thought `syncNetlifyEnvVars` would be more appropriate here. Opinion?